Sources/StreamChatUI/MessageActionsPopup/ChatMessageActionsVC.swift

```diff
@@ -97,40 +104,62 @@ open class ChatMessageActionsVC: _ViewController, ThemeProvider {
         case nil:
             var actions: [ChatMessageActionItem] = []
 
-            if channelConfig.quotesEnabled {
+            // If a channel is not set, we fallback to using channelConfig only.
+            let canQuoteMessage = channel?.ownCapabilities.contains(.quoteMessage) ?? channelConfig.quotesEnabled
```

Comment:
Shouldn't the local channel config have bigger prio than the one from the dashboard?

Reply:
No, the ownCapapabilties is more accurate than the channelConfig. Btw, ChannelConfig is not a local config, it comes from the remote as well

Reply:
ok, might have mixed things